frontend/src/components/ChoicesFilterSection.tsx

```diff
@@ -8,23 +8,48 @@ interface Props {
   title: string;
   value: string;
   choices: Record<string, string>;
+  exclusiveChoice: boolean;
   onChange: (value: string) => void;
 }
 
-const ChoicesFilterSection = ({ title, value, choices, onChange }: Props) => {
+const ChoicesFilterSection = ({
+  title,
+  value,
+  choices,
+  exclusiveChoice,
```

Comment:
I find it slightly more natural to have the opposite property `multipleChoice` but it is probably just an opinion.

Reply:
I think it is more intuitive too, fixed